tests/cli/test_cli.py

```diff
@@ -272,13 +279,10 @@ def test_backfill(self, mock_run):
                 '-s', DEFAULT_DATE.isoformat()]), dag=dag)
 
         mock_stdout.seek(0, 0)
-        self.assertListEqual(
-            [
-                "Dry run of DAG example_bash_operator on {}\n".format(DEFAULT_DATE.isoformat()),
-                "Task runme_0\n",
-            ],
-            mock_stdout.readlines()
-        )
+
+        output = mock_stdout.read()
+        self.assertIn("Dry run of DAG example_bash_operator on {}\n".format(DEFAULT_DATE.isoformat()), output)
+        self.assertIn("Task runme_0\n".format(DEFAULT_DATE.isoformat()), output)
```

Comment:
Airflow has a side effect that will be difficult to fix, so I changed this test a bit.
The following message began to appear at the beginning of the output.
```
[2019-11-12 00:15:08,234] {cli_action_loggers.py:67} DEBUG - Calling callbacks: [<function default_action_log at 0x7f37bf1aed08>]
```